### DISCUSSION TITLE
Are dependencies downgrades correct?

### DISCUSSION THREAD

**Discussion by mdesantis** [ElMassimo/vite_ruby#166] [Q&A] [answered]
When I run `bundle vite upgrade`, the packages versions are downgraded to:

```
    "vite": "^2.6.13",
    "vite-plugin-ruby": "^3.0.2"
```

But the latest packages versions are:

```
    "vite": "^2.7.7",
    "vite-plugin-ruby": "^3.0.5"
```

Is the downgrade correct?

**Comment by ElMassimo** [comment, marked as answer]
Hi Maurizio!

In order to ensure compatibility between Vite and Vite Ruby, the `upgrade` command will follow [the version specified by Vite Ruby](https://github.com/ElMassimo/vite_ruby/blob/main/vite_ruby/lib/vite_ruby/version.rb#L7-L8).

The `vite-plugin-ruby ^3.0.2` version will be resolved to `^3.0.5` (check the lockfile), but `vite` will not be upgraded to `2.7` to prevent incompatibilities.

Although it's a bit more work on my end, locking `vite` to a minor version ensures users don't upgrade to a newer version of Vite that hasn't been tested with Vite Ruby. I'll bump it to `2.7` and release a new version of `vite_ruby`.

**Reply by ElMassimo** [reply]
Released `vite_ruby@3.0.7` which will upgrade `vite` to `2.7.7`.
